docs/contributors/code/coding-guidelines.md

```diff
@@ -238,14 +238,14 @@ export function toggleFeature( scope, featureName ) {
 }
 ```
 
-#### Use the `lock()` and `unlock()` API from `@wordpress/experiments` to privately export almost anything
+#### Use the `lock()` and `unlock()` API from `@wordpress/private-apis` to privately export almost anything
 
 Each `@wordpress` package wanting to privately access or expose experimental APIs can
-do so by opting-in to `@wordpress/experiments`:
+do so by opting-in to `@wordpress/private-apis`:
 
 ```js
-// In packages/block-editor/experiments.js:
-import { __dangerousOptInToUnstableAPIsOnlyForCoreModules } from '@wordpress/experiments';
+// In packages/block-editorwordpress/private-apis.js:
```

Comment:
@adamziel Sorry for the post-merge pick up, should this line read `packages/block-editor/private-apis.js`

I noticed it elsewhere in the PR too.

Reply:
Thanks! https://github.com/WordPress/gutenberg/pull/47953